README.md

```diff
@@ -101,6 +101,63 @@ Check out the [API docs](http://registry.origami.ft.com/components/o-share#docs-
 
 ## Migration guide
 
+### Migrating from v5.x.x to v6.x.x
+
+o-share v6 introduces a breaking change that you may need to update in your product:
+
+It may help to look at the [changes made to the demo markup](https://github.com/Financial-Times/o-share/pull/100/commits/a7bb5de62d16bd4b4610d80e1c863e32335bf548#diff-bf0fb135efa69a14d71d3a973a919ad5)
+
+- CSS type selectors that were used to apply styles to the `<i>` tag have been removed so an additional class is required.
+- It is recommended you also use a semantically correct tag instead of the `<i>`.
```

Comment:
~It is recommended~ *We recommend* you also use a semantically correct tag instead of the `<i>`.

We try to avoid the passive voice in our documentation (https://github.com/Financial-Times/ft-origami#documentation-style-guide - http://www.hemingwayapp.com/ helps with this if you want to know more.)

Reply:
Amazing I wasn't aware of either of this so they are probably going to save me lots of headaches in the future. Thanks